pages/common/kubectl.md

```diff
@@ -1,13 +1,9 @@
 # kubectl
 
 > Command line interface for running commands against Kubernetes clusters.
-> More information: <https://kubernetes.io/docs/tasks/tools/install-kubectl>.
+> More information: <https://kubernetes.io/docs/reference/kubectl/>.
 
-- List all pods in all namespaces:
-
-`kubectl get pods --all-namespaces`
-
-- List all pods with more information (such as node name):
+- List all information about a resource (pods, deployments, services, ingresses, etc.) with more details:
 
 `kubectl get pods -o wide`
```

Comment:
In this case, if we want to put a generic object instead of a pod (which I think is a good suggestion), we would need to change pods with `{{pod|deployment|services|ingress|...}}`.

@agnivade can you just help us with what is the right syntax to use in this case: if we want to specify multiple objects in `{{}}` but there are a potentially unlimited number of them. Is it ok to put `|...` in the end?

Reply:
Yes that is correct. See the page for tshark. Although we are using the unicode ellipsis there instead of 3 dots. I think we later reverted our decision to use 3 dots again instead of ellipsis. Does anyone remember ? @sbrl @mebeim @waldyrious 

btw - the example has pods, so is saying `pod` equivalent to `pods` ?

Reply:
Yeah, I do believe we decided to use ASCII characters for maximum compatibility, though I can't remember the ultimate reason.

Reply:
@agnivade `pods` & `pod` are the equivalents. In this example I would use singular.

Reply:
> I think we later reverted our decision to use 3 dots again instead of ellipsis. Does anyone remember ? `@sbrl` `@mebeim` `@waldyrious`

> Yeah, I do believe we decided to use ASCII characters for maximum compatibility, though I can't remember the ultimate reason.

We actually never wrapped the discussion. The first change to use the ellipsis character was done in #2745, and the unfinished discussion about whether to use ASCII or Unicode is [here](https://github.com/tldr-pages/tldr/pull/2808#discussion_r261263925). Maybe let's try to finalize the discussion there?

Reply:
Agree. I suggest to revert back to 3 dots.

Reply:
I'm fine with switching to ASCII 3 dots.